IntegrationTests/Services/AWSS3IntegrationTests/S3EmptyBody404Tests.swift

```diff
@@ -38,7 +38,7 @@ class S3EmptyBody404Tests: S3XCTestCase {
         } catch let error as AWSServiceError & ClientRuntime.HTTPError where error.errorCode == "NotFound" {
 
```

Comment:
Replace `XCTAssertEqual` because that requires Equatable conformance. Use `.isEmpty()` instance method instead.